db/models.py

```diff
@@ -1 +1,30 @@
 from django.db import models
+
+
+class Race(models.Model):
+
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+
+    name = models.CharField(max_length=255, unique=True)
+    description = models.CharField(max_length=255, null=True)
+
+
+class Player(models.Model):
+
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=False)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
Do not forget tot add `related_name` as mentioned [in the checklist](https://github.com/mate-academy/py-game-models/blob/master/checklist.md#3-do-not-forget-to-add-related_name-to-foreignkey-fields) in all places